assets/xml/scenes/misc/hiral_demo.xml

```diff
@@ -4,6 +4,16 @@
         <Scene Name="hiral_demo_scene_unused1" Offset="0x3580"/>
         <Scene Name="hiral_demo_scene_unused2" Offset="0x3640"/>
         <Scene Name="hiral_demo_scene_unused3" Offset="0x3700"/>
+
+        <Cutscene Name="gCutsceneMapKokiriEmeraldPart2Cs" Offset="0x0E50"/> <!-- Ganondorf riding the horse -->
+        <Cutscene Name="gCutsceneMapKokiriEmeraldPart3Cs" Offset="0x0100"/> <!-- World creation part 1 -->
+        <Cutscene Name="gCutsceneMapKokiriEmeraldPart8Cs" Offset="0x1260"/> <!-- Triforce creation -->
+        <Cutscene Name="gCutsceneMapGanondorfFromMasterSwordUnusedCs" Offset="0x1A60"/> <!-- Unused Ganondorf cutscene from the Master Sword -->
+        <Cutscene Name="gCutsceneMapTriforceSheikahLegendCs" Offset="0x1F00"/> <!-- The Sheikah legend about the Triforce -->
+        <Cutscene Name="gCutsceneMapDekuSproutPart2Cs" Offset="0x2310"/> <!-- The Deku Tree Sprout's story -->
+        <Cutscene Name="gCutsceneMapGanonHorseCs" Offset="0x2490"/> <!-- Used for the title screen -->
```

Comment:
```suggestion
        <Cutscene Name="gCutsceneMapTitleGanonHorseCs" Offset="0x2490"/> <!-- Used for the title screen -->
```

The name should express that it's part of the title screen set of cutscenes, since it (almost) always branches back to the Title Screen